src/app/service/httpService.js

```diff
@@ -1,60 +1,55 @@
 import axios from 'axios';
 
-//You can setup default config right here
+// You can setup default config right here
 
 const defaultConfig = {
   headers: {
-    'X-Custom-Header': 'foobar'
-  }
+    'X-Custom-Header': 'foobar',
+  },
 };
 
 export default class httpService {
-
   // If no config will be passed to the class arguments, the defaultConfig will be used
 
-  constructor(config = defaultConfig){
+  constructor(config = defaultConfig) {
     this.config = config;
   }
 
-  async get(url){
-      try {
+  async get(url) {
+    try {
       const response = await axios.get(url, this.config);
       return response.data;
-    }
-    catch (error) {
+    } catch (error) {
       return error.response.data;
     }
   }
 
-  async post(url, data){
+  async post(url, data) {
     try {
       const response = await axios.post(url, data, this.config);
       return response.data;
-    }
-    catch (error) {
+    } catch (error) {
       return error.response.data;
     }
   }
 
-  async put(url, data){
+  async put(url, data) {
     try {
       const response = await axios.put(url, data, this.config);
       return response.data;
-    }
-    catch (error) {
+    } catch (error) {
       return error.response.data;
     }
   }
 
-  async delete(url){
+  async delete(url) {
     try {
       const response = await axios.delete(url, this.config);
       return response.data;
-    }
-    catch (error) {
+    } catch (error) {
       return error.response.data;
     }
   }
 };
 
- 
\ No newline at end of file
+
```

Comment:
I don't like this approach to commit unrelated files in one PR

Reply:
my bad, I will redo the PR only with files that are related to my task

Reply:
@Hollister009 The thing is if you pull development right now - you can't push it without fixing everything. 3 PR's where merged when we had no eslint/hooks/ etc.

Reply:
@Hollister009 I created a new PR https://github.com/Nazar-bnm/ui-internship/pull/6, created a new branch from development, added files from my task only and pushed it with ```--no-verif```. If it's suitable for you i will close this PR